src/commands/build.ts

```diff
@@ -131,10 +168,60 @@ export default class Build extends BaseCommand {
       );
     }
 
+    if (flags.path) {
+      let trackingPlans = [] 
+      for (const plan of configPlans) {
+        const oldPath = './' + plan.path
+        const configPlan = await loadTrackingPlan(this.configPath, plan)
+        const { path } = await this.prompt({
+          type: "input",
+          name: "path",
+          message: this.formatQuestion(
+            `Enter a directory for the Tracking Plan ${chalk.green(
+              configPlan?.name
+            )} output:`,
+            [
+              "The generated client will be stored in this directory.",
+              "Directories will be automatically created, if needed.",
+            ]
+          ),
+          filter: normalize,
+          default: plan?.path
+        });
```

Comment:
I don't think we should prompt on the `build` command as most of the times these commands are run in non-interactive CI environments.